homework-projects/rupesh_final/template-included-file-1.adoc

```diff
@@ -0,0 +1,35 @@
+[[included-file-template-1]]
+= Using Peek: A Step-by-Step Guide +
+
+Replace me: This is a included file template #1 with a sample cross reference to xref:template-included-file-2.adoc[Included file template #2]. 

```

Comment:
You can replace the replace me part with your own content.

Reply:
I just made it short as the content is the same